pkg/handler3/handler_test.go

```diff
@@ -39,7 +39,7 @@ var returnedOpenAPI = []byte(`{
    "title": "Kubernetes",
    "version": "v1.23.0"
   },
-  "paths": {}}`)
+  "paths": {"x-kubernetes-extension": "val"}}`)
```

Comment:
Why is that happening?

Reply:
agreed... what's this about?

Reply:
Removed. There was a bug with the round tripping for paths earlier.